serving/docs/configurations_large_model_inference_containers.md

```diff
@@ -0,0 +1,54 @@
+| option.model_id         | No       | The value of this option will be the Hugging Face ID of a model or the s3 url of the model artifacts. DJL Serving will use the ID to download the model from Hugging Face or the s3 url. DJL Serving uses `s5cmd` to download the model from the bucket which is generally faster | `google/flan-t5-xl`, `s3://<my-bucket>/google/flan-t5-xl` Default: None |
+| option.enable_streaming | No       | Enables response streaming. Use `huggingface` to enable HuggingFace like streaming output                                                                                                                                                                                         | `false`, `true`, `huggingface`                                          |
+| option.dtype            | No       | Datatype to which you plan to cast the model default.                                                                                                                                                                                                                             | `fp16, fp32, bf16, int8`                                                |
+
+### DeepSpeed ([doc](https://docs.aws.amazon.com/sagemaker/latest/dg/large-model-inference-configuration.html))
+
+| Item                      | Required | Description                                                                                                                                                                                     | Example value                  |
+|---------------------------|----------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------|
+| option.task               | No       | The task used in Hugging Face for different pipelines.                                                                                                                                          | `text-generation`              |
+| option.max_tokens         | No       | Total number of tokens (input and output) with which DeepSpeed can work. The number of output tokens in the difference between the total number of tokens and the number of input tokens.       | `1024`                         |
+| option.low_cpu_mem_usage  | No       | Reduce CPU memory usage when loading models. We recommend that you set this to True.                                                                                                            | `True`                         |
```

Comment:
Let use lower case for all True/False value as example in the serving.properties

Reply:
Done